apps/snitch_core/test/data/schema/country_test.exs

```diff
@@ -0,0 +1,113 @@
+defmodule Snitch.Data.Schema.CountryTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+
+  alias Snitch.Repo
+  alias Snitch.Data.Schema.Country
+
+  @valid_attrs %{
+    iso_name: "INDIA",
+    iso: "IN",
+    iso3: "IND",
+    name: "India",
+    numcode: "356"
+  }
+
+  @db_attrs %Country{
+    iso_name: "INDIA",
+    iso: "IN",
+    iso3: "IND",
+    name: "India",
+    numcode: "356"
+  }
+
+  describe "Countries" do
+    test "with valid attributes" do
+      %{valid?: validity} = Country.changeset(%Country{}, @valid_attrs)
+      assert validity
+    end
+
+    test "with invalid attributes" do
+      params = Map.delete(@valid_attrs, :numcode)
+      %{valid?: validity} = Country.changeset(%Country{}, params)
+      refute validity
+    end
+
+    test "with invalid iso" do
+      params = Map.update!(@valid_attrs, :iso, fn _ -> "IND" end)
```

Comment:
Why don't you use `Map.put` instead?

Reply:
Here there is no difference using Map.put or Map.update!